mesonbuild/interpreter/type_checking.py

```diff
@@ -174,6 +174,8 @@ def variables_convertor(contents: T.Union[str, T.List[str], T.Dict[str, str]]) -
     convertor=_install_mode_convertor,
 )
 
+INSTALL_NAME_KW: KwargInfo[T.Optional[str]] = KwargInfo('install_name', (str, NoneType))
```

Comment:
Since all use cases have the same since= value it may make sense to include it here rather than always evolving it.

Reply:
Thanks, I must have missed the other cases using this pattern. Fixed.